Applications/Spire/Include/Spire/Charting/StyleDropDownMenuItem.hpp

```diff
@@ -0,0 +1,51 @@
+#ifndef SPIRE_STYLE_DROPDOWN_MENU_ITEM_HPP
+#define SPIRE_STYLE_DROPDOWN_MENU_ITEM_HPP
+#include <QWidget>
+#include "Spire/Charting/TrendLine.hpp"
+#include "Spire/Ui/Ui.hpp"
+
+namespace Spire {
+
+  //! Represents a style item in a StyleDropDownMenu.
+  class StyleDropDownMenuItem : public QWidget {
+    public:
+
+      //! Signals that this item was selected.
+      using SelectedSignal = Signal<void (TrendLineStyle style)>;
```

Comment:
What is "this" item? Document the parameters to the slot.

Reply:
What would be better? That 'an instance of this class was selected'?

Reply:
Signals a user selected a trend line.
param style - The trend line selected.